app/main.py

```diff
@@ -1,6 +1,29 @@
+import os
+from datetime import datetime as dt
+
+from dotenv import load_dotenv
+import requests
+
+load_dotenv()
+BASE_URL = "https://api.weatherapi.com/v1/current.json?"
+CITY = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    api_key = os.getenv("API_KEY")
+    if not api_key:
+        raise ValueError("No API key found in environment variables")
+
+    url = BASE_URL + f"key={api_key}&q={CITY}"
```

Comment:
You've done well by using a constant for the base URL. However, you should also use constants for the other parts of your URL. This includes the 'key=' and 'q=' parts. This will make your code more maintainable and easier to read.